guides/micronaut-mongodb-asynchronous/java/src/test/java/example/micronaut/FruitControllerTest.java

```diff
@@ -0,0 +1,44 @@
+package example.micronaut;
+
+import io.micronaut.context.ApplicationContext;
+import io.micronaut.http.HttpStatus;
+import io.micronaut.runtime.server.EmbeddedServer;
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import java.util.Collections;
+import java.util.List;
+
+public class FruitControllerTest {
```

Comment:
Does it make sense to share this template with the synchronous guide since they should be the same

Reply:
We don't have support for sharing code between guides.